sdk/core/azure-core/src/main/java/com/azure/core/util/CoreUtils.java

```diff
@@ -197,7 +197,12 @@ public static <T> Publisher<T> extractAndFetch(PagedResponse<T> page, Context co
      * @return an immutable {@link Map}.
      */
     public static Map<String, String> getProperties(String propertiesFileName) {
-        try (InputStream inputStream = CoreUtils.class.getClassLoader().getResourceAsStream(propertiesFileName)) {
+        InputStream inputStream = null;
```

Comment:
If we couldn't find the resource (propertiesFileName), we should throw some exception instead of silently failing and returning empty map. Let the client library handle the exception.